admin-manual/installation/linux/ubuntu-bionic.rst

```diff
@@ -364,56 +288,50 @@ called :file:`/etc/php5/fpm/pool.d/atom.conf`:
    php_admin_value[html_errors] = off
    php_admin_value[session.use_only_cookies] = 0
 
-   # APC, which is still used in PHP 5.5 for userland memory cache unless you
-   # are switching to something like sfMemcacheCache
+   ; APC
    php_admin_value[apc.enabled] = 1
    php_admin_value[apc.shm_size] = 64M
    php_admin_value[apc.num_files_hint] = 5000
    php_admin_value[apc.stat] = 0
 
-   # Zend OPcache
-   # Only in Ubuntu 14.04 (PHP 5.5).
-   # Don't use this in Ubuntu 12.04, it won't work.
+   ; Zend OPcache
    php_admin_value[opcache.enable] = 1
-   php_admin_value[opcache.enable_cli] = 0
    php_admin_value[opcache.memory_consumption] = 192
    php_admin_value[opcache.interned_strings_buffer] = 16
    php_admin_value[opcache.max_accelerated_files] = 4000
    php_admin_value[opcache.validate_timestamps] = 0
    php_admin_value[opcache.fast_shutdown] = 1
 
-   # This is a good place to define some environment variables, e.g. use
-   # ATOM_DEBUG_IP to define a list of IP addresses with full access to the
-   # debug frontend or ATOM_READ_ONLY if you want AtoM to prevent
-   # authenticated users
+   ; This is a good place to define some environment variables, e.g. use
+   ; ATOM_DEBUG_IP to define a list of IP addresses with full access to the
+   ; debug frontend or ATOM_READ_ONLY if you want AtoM to prevent
+   ; authenticated users
    env[ATOM_DEBUG_IP] = "10.10.10.10,127.0.0.1"
    env[ATOM_READ_ONLY] = "off"
 
-Note that the section "Zend OPcache" won't work in Ubuntu 12.04. Comment it out
-or remove it unless you are using Ubuntu 14.04.
-
-The process manager has to be restarted:
+The process manager has to be enabled and restarted:
 
 .. code-block:: bash
 
-   sudo service php5-fpm restart
+   sudo systemctl enable php7.2-fpm
+   sudo systemctl start php7.2-fpm
 
 If the service fails to start, make sure that the configuration file has been
 has been pasted properly. You can also check the syntax by running:
 
 .. code-block:: bash
 
-   sudo php5-fpm --test
+   sudo php-fpm7.2 --test
```

Comment:
`php7.2-fpm`

Reply:
Seems like the command is named `php-fpm7.2 ` :man_shrugging:

Reply:
:man_facepalming: